app/assets/stylesheets/user.css

```diff
@@ -0,0 +1,163 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;500;600;700;800&family=Poppins&family=Roboto:wght@700&display=swap');
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'poppins', sans-serif;
+}
+
+/* FOR USERS/INDEX */
+.user-container {
+  padding: 3rem 6rem;
+  background-color: #f4f4f4;
+}
+
+.user-wrapper {
+  background-color: #fff;
+  border-radius: 10px;
+  padding: 1rem;
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+  gap: 0;
+}
+
+.user-image {
+  width: 35vh;
+}
+
+a {
+  text-decoration: none;
+}
+
+.username a {
+  font-size: 40px;
+  text-decoration: none;
+  color: #282828;
+  font-family: 'poppins', sans-serif;
+}
+
+.btn a {
+  font-weight: 700;
+  color: rgb(0, 0, 0);
+  border-radius: 5px;
+  padding: 10px;
+  margin: 50px auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.post-btn a {
+  font-size: 12px;
+  color: #fff;
+  background-color: rgb(179, 65, 65);
+  border-radius: 5px;
+  padding: 10px;
+}
+
+.link:hover a {
+  color: antiquewhite;
+  background-color: rgb(150, 52, 52);
+  padding: 10px;
+  border-radius: 8px;
+  transition: all 0.5s ease-out 0.1s;
+}
+
+.user-details {
+  display: flex;
```

Comment:
- [ ] Great job so far to be consistent with the wireframe let's add the borders.

### Wireframe
<img width="266" alt="image" src="https://user-images.githubusercontent.com/59416136/222802270-66b61f34-71e8-4f6d-a4d8-67c52a7665b2.png">

### Your Implementation
<img width="562" alt="image" src="https://user-images.githubusercontent.com/59416136/222802480-8881f887-81d2-4a7f-a160-ab2f2390f871.png">